src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/HazmatSuitListener.java

```diff
@@ -0,0 +1,59 @@
+package io.github.thebusybiscuit.slimefun4.implementation.listeners;
+
+import org.bukkit.entity.Bee;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.entity.EntityDamageByEntityEvent;
+import org.bukkit.inventory.ItemStack;
+
+import io.github.thebusybiscuit.cscorelib2.inventory.ItemUtils;
+import io.github.thebusybiscuit.slimefun4.api.MinecraftVersion;
+import me.mrCookieSlime.Slimefun.Objects.SlimefunItem.SlimefunItem;
+import me.mrCookieSlime.Slimefun.SlimefunPlugin;
+
+/**
+ * The listener for Hazmat Suit's {@link Bee} sting protection.
+ * Only applied if the whole set is worn.
+ *
+ * @author Linox
+ *
+ * @see Bee
+ *
+ */
+public class HazmatSuitListener implements Listener {
+
+    public HazmatSuitListener(SlimefunPlugin plugin) {
+        plugin.getServer().getPluginManager().registerEvents(this, plugin);
+    }
+
+    @EventHandler
+    public void onDamage(EntityDamageByEntityEvent e) {
+        if (!SlimefunPlugin.getMinecraftVersion().isAtLeast(MinecraftVersion.MINECRAFT_1_15)) return;
```

Comment:
Performance-wise this is a rather careless design choice. You should put this if-statement in front of initializing this Listener in the first place (in SlimefunPlugin.java), otherwise you are registering a completely unnecessary Listener.

And let's be honest... the Minecraft version won't change from one minute to another.